src/backend/SyncBackendConnection.cpp

```diff
@@ -48,21 +48,21 @@ ListItem* SyncBackendConnection::create_process_data_(
         EntityId id)
 {
     qDebug() << "Creating Process " << backend::backend_id_to_models_id(id);
-    return new ProcessModelItem(id, EntityKind::HOST, get_info(id));
+    return new ProcessModelItem(id, EntityKind::PROCESS, get_info(id));
```

Comment:
NIT;
Rearrange these methods so the follow the same order as the rest of the whole project:
Host, User, Process
(This may have been the source of this error)